PowerVCF.psm1

```diff
@@ -6327,3 +6327,600 @@ Function Debug-CatchWriter {
 Export-ModuleMember -Function Debug-CatchWriter
 
 #EndRegion Useful Script Functions
+
+
+#Region JSON Export Functions
+Function Export-VCFManagementDomainJsonSpec {
+
+    Param (
+        [Parameter (Mandatory = $true)] [String]$workbook,
+        [Parameter (Mandatory = $true)] [String]$jsonPath
+    )
+    # Confirm the presence of ImportExcel module
+    If (!(Get-InstalledModule -name "ImportExcel"  -MinimumVersion 7.8.5 -ErrorAction SilentlyContinue)) {
+        Write-Host " ImportExcel PowerShell module not found. Please install manually" -ForegroundColor Yellow
```

Comment:
Do you think we should use `Write-Output` rather than `Write-Host`?
This is what we do in PowerValidatedSolutions and all other modules so that we can capture the output and reuse if required. Write-Host is specific to the console as far as I'm aware.

Reply:
Done..cheers!